src/app/dashboard/reports/page.tsx

```
@@ -50,123 +50,116 @@ export default function ReportsDashboard() {
-                    <span className="px-2 inline-flex text-xs leading-5 font-semibold rounded-full bg-green-100 text-green-800">
+                  <td className="px-4 py-4 whitespace-nowrap text-gray-700">{report.type}</td>
+                  <td className="px-4 py-4 whitespace-nowrap text-gray-500">{report.date}</td>
+                  <td className="px-4 py-4 whitespace-nowrap">
+                    <span className="px-2 inline-flex text-xs font-semibold rounded-full bg-green-100 text-green-700">
                       {report.status}
                     </span>
                   </td>
-                  <td className="px-6 py-4 whitespace-nowrap text-sm font-medium">
-                    <a href="#" className="text-indigo-600 hover:text-indigo-900 mr-3">
+                  <td className="px-4 py-4 whitespace-nowrap space-x-2">
+                    <a href="#" className="text-indigo-600 hover:text-indigo-900">
```

Biome (2.1.2)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)